src/test/java/spoon/test/comment/CommentTest.java

```diff
@@ -110,12 +111,37 @@ public void testCombinedPackageInfoComment() {
 
 	@Test
 	public void testJavaDocComment() {
-		String EOL = System.getProperty("line.separator");
+		//the EOL is taken from JavaDocComment.java, which is commited in git with linux \n
+		//that is true on Windows too.
+		String EOL = "\n";
 
 		Factory f = getSpoonFactory();
 		CtClass<?> type = (CtClass<?>) f.Type().get(JavaDocComment.class);
 
 		CtJavaDoc classJavaDoc = (CtJavaDoc) type.getComments().get(0);
+		//contract: test that java doc is printed correctly
+		String str = classJavaDoc.toString();
+		StringTokenizer st = new StringTokenizer(str, "\n\r");
```

Comment:
Ok for the behaviour of StringTokenizer but I'm not sure to understand why you want to do that here. If you're on linux, you shouldn't have `\r` at all, so using `System.getProperty("line.separator")` would be safer, and if you're on windows it will return `\n\r` so you're test remains the same. In either case, it seems safer to me. What am I missing?